cmd/authctl/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/ubuntu/authd/cmd/authctl/user"
+)
+
+const cmdName = "authctl"
+
+var rootCmd = &cobra.Command{
+	Use:   fmt.Sprintf("%s COMMAND", cmdName),
+	Short: "CLI tool to interact with authd services",
+	Long:  "authctl is a CLI tool which can be used to interact with various authd services.",
+	Args:  cobra.NoArgs,
+	Run:   func(cmd *cobra.Command, args []string) {},
+}
+
+func init() {
+	rootCmd.AddCommand(user.UserCmd)
+}
+
+func main() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Fprintf(os.Stderr, "Error while executing authctl '%s'\n", err)
```

Comment:
"Error while executing authctl" doesn't provide any useful info. Let's just print the error.

```suggestion
		fmt.Fprintln(os.Stderr, err.Error())
```

Reply:
Done.